01.general/17.dns-providers/docs.en.md

```diff
@@ -362,6 +535,180 @@ Regular and DNS-over-TLS servers which provide protection from phishing and spyw
 | DNS, IPv4      | `185.121.177.177` and `169.239.202.202`            | <a href="sdns://AAAAAAAAAAAADzE4NS4xMjEuMTc3LjE3Nw">Add to AdGuard</a> |
 | DNS, IPv6      | `2a05:dfc7:5::53` and `2a05:dfc7:5353::53`         | <a href="sdns://AAAAAAAAAAAAEVsyYTA1OmRmYzc6NTo6NTNd">Add to AdGuard</a> |
 
+<a name="blahdns.ger-dns"></a>
+
+### BlahDNS
+
+[BlahDNS](https://blahdns.com/) A small hobby DNS project. No logs, Ethereum Name Service, DNSSEC ready & Filtered ads, trackers, malwares
+
+#### Finland DNS Server
+
+| Protocol       | Address                             |                |
+|----------------|-------------------------------------|----------------|
+| DNS-over-TLS, IPv4 | Hostname: `tls://dot-fi.blahdns.com`  IP: `95.216.212.177` | <a href="sdns://AwAAAAAAAAAAAAAYdGxzOi8vZG90LWZpLmJsYWhkbnMuY29t">Add to AdGuard</a> |
+| DNS-over-HTTPS, IPv4 | Hostname: `https://doh-fi.blahdns.com/dns-query` IP: `95.216.212.177` | <a href="sdns://AgMAAAAAAAAAAAASZG9oLWRlLmJsYWhkbnMuY29tCi9kbnMtcXVlcnk">Add to AdGuard</a> |
```

Comment:
This stamp is for `doh-de.blahdns.com` :)
DNSCrypt stamps are also incorrect

Reply:
Updated.